main/core/messagebroker.ts

```diff
@@ -99,6 +102,21 @@ export class MessageBroker<T = any> implements IMessageBroker<T> {
         delete this.channelLookup[channelName];
     }
 
+    /**
+     * Creates a new scope with the given scopeName with this instance of the MessageBroker as its parent.
+     * If a scope with this name already exists, it returns that instance instead of creating a new one.
+     * @param scopeName The name to use for the scope to create
+     * @returns An instance of the messagebroker that matches the scopeName provided
+     */
+    public createScope(scopeName: string): IMessageBroker<T> {
+        const scope = rootInjector.createScope(scopeName);
+        scope.registerInstance(MessageBroker, new MessageBroker<T>(new RSVPMediator(), this));
```

Comment:
So we will have a scoped RSVPMediator too.  We ok with that? What does that mean?  Is it still global in nature under the covers?